lib/phoenix_live_view/html_formatter.ex

```diff
@@ -225,6 +236,21 @@ defmodule Phoenix.LiveView.HTMLFormatter do
       line_length = opts[:heex_line_length] || opts[:line_length] || @default_line_length
       newlines = :binary.matches(source, ["\r\n", "\n"])
 
+      opts =
+        Keyword.update(opts, :attribute_formatters, %{}, fn formatters ->
+          {_, formatters} =
+            Enum.flat_map_reduce(formatters, %{}, fn {attr, formatter}, formatters ->
```

Comment:
It seems a `Enum.reduce` would suffice here!

Reply:
Oh, right! :sweat_smile: Thank you!